app/src/main/java/com/xzyht/notifyrelay/sync/MessageSender.kt

```
@@ -536,12 +243,82 @@ object MessageSender {
+                                val uri = Uri.parse(v)
+                                context.contentResolver.openInputStream(uri)?.use { input ->
+                                    val bytes = input.readBytes()
+                                    val mime =
+                                        context.contentResolver.getType(uri) ?: "image/png"
+                                    val b64 = Base64.encodeToString(bytes, Base64.NO_WRAP)
+                                    processedPics[k] = "data:$mime;base64,$b64"
+                                } ?: run {
+                                    // 无法打开则回退到原始字符串
+                                    processedPics[k] = v
+                                }
+                            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)